apps/web/pages/organizations/[organizationId]/invoices/[invoiceId]/components/invoiceDetails.tsx

```diff
@@ -36,72 +36,72 @@ export const InvoiceDetails = ({ invoice, invoiceItems }: InvoiceDetailsProps) =
         </div>
       </div>
 
-      <table className="size-full border-collapse border border-gray-200">
-        <thead className="bg-gray-200 text-left text-sm">
+      <table className="table size-full border-collapse border border-neutral">
+        <thead className="bg-neutral text-sm text-neutral-content">
           <tr>
-            <th className="border border-gray-300 px-4 py-2">Item</th>
-            <th className="border border-gray-300 px-4 py-2">Duration</th>
-            <th className="border border-gray-300 px-4 py-2">Hourly Rate</th>
-            <th className="border border-gray-300 px-4 py-2">Amount</th>
+            <th>Item</th>
+            <th>Duration</th>
+            <th>Hourly Rate</th>
+            <th>Amount</th>
           </tr>
         </thead>
         <tbody>
           {invoiceItemsData.map((invoiceItem) => (
             <tr key={invoiceItem.id}>
-              <td className="border border-gray-200 px-4 py-2">{invoiceItem.task.title}</td>
-              <td className="border border-gray-200 px-4 py-2">{invoiceItem.duration}</td>
-              <td className="border border-gray-200 px-4 py-2">{invoiceItem.hourlyRate}</td>
-              <td className="border border-gray-200 px-4 py-2">{invoiceItem.duration * invoiceItem.hourlyRate}</td>
+              <td className="border border-neutral">{invoiceItem.task.title}</td>
+              <td className="border border-neutral">{invoiceItem.duration}</td>
+              <td className="border border-neutral">{invoiceItem.hourlyRate}</td>
+              <td className="border border-neutral">{invoiceItem.duration * invoiceItem.hourlyRate}</td>
             </tr>
           ))}
-          <tr>
-            <td className="border border-gray-200 px-4 py-2" colSpan={2} />
-            <td className="border border-gray-200 px-4 py-2 font-bold">Total</td>
-            <td className="border border-gray-200 px-4 py-2 font-bold">
+          <tr className="font-bold">
+            <td colSpan={2} className="border border-neutral" />
+            <td className="border border-neutral">Total</td>
+            <td className="border border-neutral">
               €{invoiceItemsData.reduce((sum, item) => sum + item.duration * item.hourlyRate, 0)}
             </td>
           </tr>
         </tbody>
       </table>
-      <div className="flex">
-        <div className="flex flex-col">
-          <p className="text-sm font-bold">
-            Payment method: <span className="font-normal">Bank Transfer/ PayPal</span>
-          </p>
-          <p className="text-sm">Thank you for your business!</p>
-        </div>
-        <div className="w-full text-right font-bold">
-          Invoice status: <span className="font-normal">{invoiceData.invoiceStatus}</span>
-        </div>
+      <div className="text-sm">
+        <p className="font-bold">
+          Payment method: <span className="font-normal">Bank Transfer / PayPal</span>
+        </p>
+        <p>Thank you for your business!</p>
```

Comment:
```suggestion
      <div className="flex justify-between">
        <div className="text-sm">
          <p className="font-bold">
            Payment method: <span className="font-normal">Bank Transfer / PayPal</span>
          </p>
          <p>Thank you for your business!</p>
        </div>
        <div className="flex gap-2 self-end">
          <Link className="btn btn-neutral btn-sm" href={`/organizations/${organizationId}`}>
            Cancel
          </Link>
          <button className="btn btn-primary btn-sm print:hidden">Send</button>
        </div>
      </div>
```

And maybe a button as well to go back to organization view page?

Reply:
![image](https://github.com/user-attachments/assets/0190aa8b-3919-4f01-b79b-bc6f12735e0d)

Reply:
Good idea. This will also be done in the other ticket.